ocdsdata/cli/commands/run.py

```diff
@@ -39,18 +38,24 @@ def configure_subparser(self, subparser):
     def run_command(self, args):
         run = []
 
-        if args.runall:
+        if args.all and args.source:
+            print("You need to either specifiy a source or use all flag, not both.")
```

Comment:
'specifiy' typo. Also, always refer to flags as e.g. `--all` to avoid confusion.